intelmq/bots/BOTS

```diff
@@ -1,56 +1,56 @@
 {
     "Collector": {
         "Abuse.ch Feodo Tracker Domains": {
-            "description": "Abuse.ch Feodo Tracker Domains is the bot responsible to get the report from source of information.",
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

On 09/08/2015 06:47 PM, Tomás Lima wrote:
>
> The description has a pattern: starts always with the name of the bot,
e.g. 'Abuse.ch Feodo Tracker Domains'
>
And is absolutely useless. Sorry, but that's my opinion on this topic.
The name is known from the key, just one line above. I assume everyone
reading this file can read both lines. Also, the file repeats the
definition of the bots types is repeated 116 times. And in the docs,
where it belongs to.

>|> Event()| constructor takes optional argument: instance of report, fields |raw|, |feed.(name|url)|,
|time.observation| are copied if existing. reduces duplicate code.

> I think we should keep with other approach. Its clear and don't have
hidden functionality. Whats your feedback?

Well, I'm for the change :) It's *exactly* the same code for *every*
parser. It reduces code in the bots, the the attention of the reader
(dev) is focused on the relevant parts, i.e. all the data handling
except those just copied - they are not relevant for the parser code.
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2.0.22 (GNU/Linux)

iQIcBAEBCAAGBQJV7yPKAAoJEBn0X+vcm0Y7lhAQALZ3uggU8XwnD7ja3f1u5Ccc
E23BgUiYqv2nmV7d/AIYpLM7giqWo0MZ8F8aoZU0J+Z18bh9Pg/Noe68AZ+zs/Ld
5qtO1gqinCNcTQFSu0C7qFpfi8Vmpyh/8Xz7i1wDwISH4LTNq0F6ZulW4ISTL80g
W9VDXf8DCl0X1uRyvdkqw0D3WBKPjXKHaHu5QM9SgWFLmseNu77CYPCUYHa2WHG0
5nIbuIda5bhPR6msVutW/t5QWd652EanKGDYoxygdHmOK4epTiOHmM3b8+vcvIQs
7oQFv/noACf0Dh2/UjXGNsJWPq8gi2xYInPYC9SbIU/dgoh+B9GvY+ckMIyenRDk
jPlntREnrXGYQS8Kj1If5oWTC9bQJasZlXPvP/M/sX7V/xdocHxHlUGuce1+fk6V
c4oS8/FN6pRpjEning18E9hTUFF+moshLiEBf9QkDEGYewlU8RpW8oJYEGsvPtpd
PGyKV8zjPkw//VolGjGUmwVC1ML/v7pkoU3kH5IWpKMRp2eRdGz3rGVn5OBNDZbr
ABA6Qy2DmLytWc7ZyBTQeeCYB+ytmRJq5TFvHxDs4ICZWmCdg3PJkDhrp7EK5O5o
YCrvrHx4WpRuOVynLtiUrCnD3laXklfjA/vhiBRBKlqjeoySN5xwy4KPozkopLrE
AJNzQhNVCYBiB/Xcllox
=WCGC
-----END PGP SIGNATURE-----